### DISCUSSION TITLE
Going menu bar only

### DISCUSSION THREAD

**Discussion by ttscoff** [ttscoff/bunch-docs#40] [Q&A] [answered]
I've added a lot of features to the next update (b76) that only work in menu bar mode. Keyboard shortcuts, more modifier clicks for menu items, and other things that just can't work in Dock mode. I personally never use Dock mode anymore, and I'm curious how others are using Bunch. Would getting rid of Dock mode entirely be a pain for anybody?

**Comment by apoc527** [comment]
Kill the Dock mode. I have never once run Bunch in Dock mode and I never _will_ run Bunch in Dock mode. But maybe we're both the weirdos, so maybe wait for more feedback??

**Reply by smarj** [reply]
I almost never run _anything_ in Dock mode if it has a menubar mode where I do most of it's work. Something like DEVONthink where it _has_ a menu bar but I mostly use it in a window gets dock space. Things like Bunch which are mainly a menu, I want to be Dock-only.

**Reply by ttscoff** [reply]
Yeah, my assumption is that Bunch's target audience thinks a lot like me in that regard, where utilities don't belong in the Dock. As I ponder making it commercial, I don't want to alienate anybody, but I really think it's far superior as a menu bar app.

As an aside, my original project was just to make a Dock icon that I could right click to launch other apps. I have no idea why I thought at the time that that was how I wanted to launch apps. But that's the whole reason Bunch has a Dock mode at all. It's a vestigial tail.

**Reply by geofftaylor** [reply]
I only use menu bar mode. I haven't used Dock mode since it was the only mode.

**Reply by technovangelist** [reply]
i can't think of any reason for the dock to even exist. never really used it in 10 years or so.

**Comment by maxoriggi** [comment]
I think too that it could be removed, Bunch, as it works now, makes sense only in the menubar (I have never used it in the dock even in previous versions).
Moreover consider that it's a power user tool, so, as others have noted, the menubar is the go to choice for this kind of utilities.

**Comment by ttscoff** [comment, marked as answer]
I'm going to consider this answered. I don't think supporting Dock mode makes any real sense. I'll remove it from the next update and start working on a welcome sequence that explains better where Bunch is running so new users can figure out what's going on right away. Then to remove a TON of references to "Dock mode" and "menu bar mode" in the documentation…

**Comment by ttscoff** [comment]
Build 80 is out and officially removes menu bar support. We'll see if that inspires anyone else to join the conversation. It will probably go unnoticed, based on reactions so far.
